rasa/cli/interactive.py

```diff
@@ -93,6 +93,7 @@ def interactive(args: argparse.Namespace) -> None:
 def _set_not_required_args(args: argparse.Namespace) -> None:
     args.fixed_model_name = None
     args.store_uncompressed = False
+    args.number_of_chunks = 1
```

Comment:
looks like this will overwrite whatever the user has provided for interactive training. Not obvious to me that this is desirable

Reply:
@dakshvar22 do you know why it is done?

Reply:
Not sure, looks like something Tanja had added when implementing the CLI support.